Sources/SwiftSyntax/generated/syntaxNodes/SyntaxStmtNodes.swift

```diff
@@ -1575,8 +1575,7 @@ public struct MissingStmtSyntax: StmtSyntaxProtocol, SyntaxHashable {
     }
   }
   
-  /// A placeholder, i.e. `<#statement#>` that can be inserted into the source code to represent the missing pattern.
-  /// This token should always have `presence = .missing`.
+  /// A placeholder, i.e. `<#statement#>` that can be inserted into the source code to represent the missing pattern./// This token should always have `presence = .missing`.
```

Comment:
Seems like CodeGeneration fails to add a proper line break for this particular comment.

I tried various line break options (e.g. extra spaces) in the codegen file below without avail.

https://github.com/apple/swift-syntax/blob/5922b69d0db36e5ebdadfe7a893f00ca576cb2e6/CodeGeneration/Sources/SyntaxSupport/CommonNodes.swift#L234